src/pages/workspace/WorkspaceProfilePage.tsx

```diff
@@ -76,8 +80,8 @@ function WorkspaceProfilePage({policy, currencyList = {}, route}: WorkSpaceProfi
                             title=""
                         >
                             <Image
-                                style={StyleSheet.flatten([styles.br4, styles.wAuto, styles.h68, imageStyle])}
-                                source={WorkspaceProfile}
+                                style={StyleSheet.flatten([styles.wAuto, styles.h68, imageStyle])}
+                                source={isDarkTheme ? WorkspaceProfile : WorkspaceProfileLight}
```

Comment:
Just out of curiosity, is this how we handle all theming choices? I guess it works but that means we're not future proofing this to support more than two themes. @grgia do you have thoughts there?

Reply:
hmm that's a good point. We actually have `useThemeIllustrations` hook which I think should be used in this case.

Reply:
PR with fix - https://github.com/Expensify/App/pull/37437

Reply:
@narefyev91 I already have a fix up

Reply:
Yes that's correct- thanks @luacmartins !